scripts/keep_sync.py

```diff
@@ -171,7 +171,7 @@ def parse_points_to_gpx(run_points_data, start_time):
             {
                 "latitude": point["latitude"],
                 "longitude": point["longitude"],
-                "elevation": point["verticalAccuracy"],
+                "elevation": point["verticalAccuracy"] if "verticalAccuracy" in point.keys() else None,
```

Comment:
we need to get if verticalAccuracy attr first.
if there is no value here
We do not  add it to the dict.

Reply:
Other props may not need to be judged to exist.
```
   # future to support heart rate
    points_dict_list = []
    for point in run_points_data:
        points_dict = {
            "latitude": point["latitude"],
            "longitude": point["longitude"],
            "time": datetime.utcfromtimestamp((point["timestamp"] * 100 + start_time) / 1000),
        }
        if "verticalAccuracy" in point.keys():
            points_dict["verticalAccuracy"] = point["verticalAccuracy"]
        points_dict_list.append(points_dict)
    gpx = gpxpy.gpx.GPX()
```

Reply:
1. only in is fine 
![image](https://user-images.githubusercontent.com/15976103/97082813-02c2be80-163f-11eb-8ec5-94ed6cce16ca.png)
2. points_dict["elevation"] = point["verticalAccuracy"]

Reply:
Okay, I'll pr again.